### PR TITLE
fix: Fix Devnet initialization config

### PR DESCRIPTION
**Pull request by fmoura**
Small fix. It can make some tests fail

### CODE REVIEW

File: pkg/ethutil/ethutil_test.go [hunk 1]
```diff
@@ -62,32 +63,52 @@ func (s *EthUtilSuite) TestAddInput() {
...
+	waitGroup := sync.WaitGroup{}
+	waitGroup.Add(1)
+
+	go func() {
+		waitGroup.Done()
+		inputIndex, err := AddInput(s.ctx, s.client, s.book, s.signer, payload)
+		if err != nil {
+			errChan <- err
+			return
+		}
```
Comment (GMKrieger) [inline pkg/ethutil/ethutil_test.go:78]: You should use `!s.Nil(err)` to properly use the suite

**Review by GMKrieger — COMMENTED** [review]
There are a couple of lint problems making the CI fail, aside from that, LGTM

File: pkg/ethutil/ethutil_test.go (hunk 1 above)
Comment (fmoura) [reply pkg/ethutil/ethutil_test.go:78]: Suite functions do not work inside Go routines. We need to "send them out" to threat them

**Review by GMKrieger — DISMISSED** [review]

**Review by marcelstanley — COMMENTED** [review]
I'd suggest changing the commit message for 96c04c249c48ef66012ee2284f1e27649fcd1d40 to `chore(devnet): update default initial block`.

BTW, what's the reason for the change? Is it the update to contracts 2.0? Whatever it is, please add it to the commit details.

File: pkg/ethutil/ethutil_test.go (hunk 1 above)
Comment (marcelstanley) [reply pkg/ethutil/ethutil_test.go]: The change was removed from the PR, @GMKrieger

**Review by GMKrieger — APPROVED** [review]

**Review by vfusco — APPROVED** [review]

**Comment by fmoura** [comment]
This PR is outdated.

**Pull request closed without merging**